os_stub/spdm_device_secret_lib_sample/chal.c

```diff
@@ -0,0 +1,80 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+/** @file
```

Comment:
I would just delete this comment block from each file. It doesn't add anything and the filenames are probably descriptive enough.

Reply:
yes. agree. done.